### ISSUE TITLE
CollapseClickCell not loading

### ISSUE THREAD

**Issue by isarfaktor** [bennyguitar/CollapseClick#6]
Hello,
after doing all the steps, like you wrote, I got the message: "could not load NIB with name 'CollapseClickCell'

Did I miss something out? I don't think I have to configure one, do I?

**Comment by bennyguitar** [comment]
Is CollapseClickCell.xib added into your project?

**Comment by isarfaktor** [comment]
Hi,
I included the files by using cocoa pods. there was no collapseclickcell.xib in this podfile included. so i tried it by manually adding it. same effect there.?

**Comment by bennyguitar** [comment]
Good catch, it looks like I only included .m/.h files in the podspec. After adding it, you still get the same error?

**Comment by isarfaktor** [comment]
absolutely yes :)
hm… i downloaded your file package from github and tried to add it manually, but it does not recognize the file at all… maybe I could give it a try again, after you've added the XIB to the podfile. Then I can give you feedback again.

**Comment by bennyguitar** [comment]
The only thing I can think of, is that it may not be in the "Copy Bundle Resources" list inside of "Build Phases" when you click on the target. (located in the same place you add new frameworks)

The new podspec is updated to include the xib.

**Comment by isarfaktor** [comment]
Cool.. I give it a try and will give you feedback ASAP!

**Comment by AnasQ** [comment]
I also get this error and I installed using CocoaPods. The xib is in the "Copy Bundle Resources" list in the Pods project.

**Comment by temojin** [comment]
i still have the same problems with pod version 1.0.1.

**Comment by TimE90** [comment]
Same here! What to do?

**Comment by morteza2128** [comment]
Use this code by Storyboard 
 CollapseClickCell *cell = [[CollapseClickCell alloc] initWithFrame:CGRectMake(0, 0, 320, kCCHeaderHeight)];
    //cell = [views objectAtIndex:0];
    UIStoryboard *sto = [UIStoryboard storyboardWithName:@"MainStoryboard"
                                                  bundle: nil];

```
CustomCollapsCellViewController *col = [sto  instantiateViewControllerWithIdentifier:@"someidentifieryousetinIB"];

cell = col.collapseView;
```

 but not work and give error that cell is null in CollapseClick.m

at this codes
  // Add to DataArray & isClickedArray
        [self.isClickedArray addObject:[NSNumber numberWithBool:NO]];
        [self.dataArray addObject:cell];  <<<<<----
why @bennyguitar ???

**Comment by HLGgithub** [comment]
Same problem here.... "could not load NIB with name 'CollapseClickCell'
